emloop/hooks/benchmarking.py

```diff
@@ -0,0 +1,28 @@
+"""
+Hook for benchmarking models.
+"""
+import logging
+import numpy as np
+from typing import List
+
+from . import AbstractHook
+from ..types import TimeProfile
+
+
+class Benchmarking(AbstractHook):
+    def __init__(self, batch_size, dataset, model, output_dir, **kwargs):
+        super().__init__(**kwargs)
+        self._batch_size = batch_size
+        self._dataset = dataset
+        self._model = model
+        self._output_dir = output_dir
+
+    def after_epoch_profile(self, epoch_id: int, profile: TimeProfile, streams: List[str]):
+        """Prints average time per example."""
+        for stream_name in streams:
+            batch_times = profile.get(f'eval_batch_{stream_name}')
```

Comment:
This is incompatible with python 3.5 which is supported. Please use `String.format` instead

Reply:
also please add support of python 3.7 to `setup.py`